octobot_evaluators/evaluators/strategy_evaluator.py

```diff
@@ -126,18 +131,19 @@ async def strategy_matrix_callback(self,
                                        symbol,
                                        time_frame):
         # if this callback is from a technical evaluator: ensure strategy should be notified at this moment
-        if evaluator_type == enums.EvaluatorMatrixTypes.TA.value:
-            # ensure this time frame is within the strategy's time frames
-            if common_enums.TimeFrames(time_frame) not in self.strategy_time_frames or \
-                    not self.is_technical_evaluator_cycle_complete(matrix_id,
-                                                                   evaluator_name,
-                                                                   evaluator_type,
-                                                                   exchange_name,
-                                                                   cryptocurrency,
-                                                                   symbol,
-                                                                   time_frame):
-                # do not call the strategy
-                return
+        for full_cycle_evaluator in self.get_full_cycle_evaluator_types():
+            if evaluator_type == full_cycle_evaluator:
+                # ensure this time frame is within the strategy's time frames
+                if common_enums.TimeFrames(time_frame) not in self.strategy_time_frames or \
+                        not self.is_technical_evaluator_cycle_complete(matrix_id,
```

Comment:
I'm not sure I understand. We know here that it is a full cycle evaluator but we cannot assume that it is a technical evaluator.

Reply:
That's right, now that we can have multiple type of evaluators here, we will rename this method